### PR TITLE
Updated and renamed upgrading.rst to upgrading.md

### PR DESCRIPTION
**Pull request by sandeep280993**
Updated and renamed upgrading.rst to upgrading.md. Please review and let me know of any further changes to be done.

### CODE REVIEW

File: docs/releases/upgrading.md [hunk 1]
```diff
@@ -1,153 +1,109 @@
...
+- Deprecation warnings shown in the console output will notify you of any code changes you need to make before upgrading to the following version;
+- Some releases make database schema changes that need to be reflected on your project by running `./manage.py makemigrations` - this is liable to fail if too many schema changes happen in one go.
 
 Before upgrading to a new feature release:
 
-* Check your project's console output for any deprecation warnings, and fix them where necessary;
-* Check the new version's release notes, and the :ref:`compatible_django_python_versions` table below, for any dependencies that need upgrading first;
-* Make a backup of your database.
+- Check your project's console output for any deprecation warnings, and fix them where necessary;
+- Check the new version's release notes, and the [Compatible Django / Python versions](compatible_django_python_versions) table below, for any dependencies that need upgrading first;
```
Comment (lb-) [inline docs/releases/upgrading.md:34]:
```suggestion
- Check the new version's release notes, and the [](compatible_django_python_versions) table below, for any dependencies that need upgrading first;
```

If you want to just inherit the link's label from the linked page you can just write `[]` (with no label).

File: docs/releases/upgrading.md [hunk 2]
```diff
@@ -1,153 +1,109 @@
...
+- Make a backup of your database.
 
 To upgrade:
 
-* Update the ``wagtail`` line in your project's ``requirements.txt`` file to specify the latest patch release of the version you wish to install. For example, to upgrade to version 1.8.x, the line should read::
+- Update the `wagtail` line in your project's `requirements.txt` file to specify the latest patch release of the version you wish to install. For example, to upgrade to version 1.8.x, the line should read:
 
-    wagtail>=1.8,<1.9
+```console
+      wagtail>=1.8,<1.9
```
Comment (lb-) [inline docs/releases/upgrading.md:42]:
```suggestion
wagtail>=1.8,<1.9
```

The extra indent is not needed now.

File: docs/releases/upgrading.md [hunk 3]
```diff
@@ -1,153 +1,109 @@
...
-Remember that the JavaScript and CSS files used in the Wagtail admin may have changed between releases - if you encounter erratic behaviour on upgrading, ensure that you have cleared your browser cache. When deploying the upgrade to a production server, be sure to run ``./manage.py collectstatic`` to make the updated static files available to the web server. In production, we recommend enabling `ManifestStaticFilesStorage <https://docs.djangoproject.com/en/stable/ref/contrib/staticfiles/#manifeststaticfilesstorage>`_ in the ``STATICFILES_STORAGE`` setting - this ensures that different versions of files are assigned distinct URLs.
 
+(compatible_django_python_versions)=
 
-.. _compatible_django_python_versions:
+## Compatible Django / Python versions
 
-Compatible Django / Python versions
-===================================
+...
```
Comment (lb-) [inline docs/releases/upgrading.md:62]:
```suggestion
```

Looks like the three dots were added here by accident

File: docs/releases/upgrading.md [hunk 4]
```diff
@@ -1,153 +1,109 @@
...
-* Run:
-
-  .. code-block:: console
+- Run:
 
+``` console
       pip install -r requirements.txt
       ./manage.py makemigrations
       ./manage.py migrate
+```
```
[the remark this answers is not in the document]
Comment (sandeep280993) [reply docs/releases/upgrading.md:51]: done

File: docs/releases/upgrading.md [hunk 5]
```diff
@@ -1,153 +1,109 @@
...
 New feature releases frequently add support for newer versions of Django and Python, and drop support for older ones. We recommend always carrying out upgrades to Django and Python as a separate step from upgrading Wagtail.
 
 The compatible versions of Django and Python for each Wagtail release are:
 
-+-------------------+------------------------------+-----------------------------+
 | Wagtail release   | Compatible Django versions   | Compatible Python versions  |
-+===================+==============================+=============================+
+|-------------------| --------------------------   | --------------------------  |
 | 0.1               | 1.6                          | 2.7                         |
-+-------------------+------------------------------+-----------------------------+
```
[the remark this answers is not in the document]
Comment (sandeep280993) [reply docs/releases/upgrading.md:75]: Double-checked in the `Preview` tab and in `Dillinger.io`.

**Comment by sandeep280993** [comment]
Need guidance in hiding **(compatible_django_python_versions)=** from the file.

**Comment by lb-** [comment]
Replaced by https://github.com/wagtail/wagtail/pull/8769/

**Pull request closed without merging**